plugins/org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/parser/antlr/AntlrDebugGrammarGenerator.xtend

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) 2015 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtext.xtext.generator.parser.antlr
+
+import com.google.inject.Inject
+import org.eclipse.xtext.Grammar
+
+class AntlrDebugGrammarGenerator extends AbstractAntlrGrammarGenerator {
```

Comment:
It's in all the class names consistently. But there will be a code review of the new generator with special emphasize on naming, so we can discuss that then.